lib/dirac.js

```diff
@@ -471,6 +471,10 @@ dirac.tx = {
       tx[ tbl ] = new Dal( client );
       tx[ tbl ].befores = dirac.dals[ tbl ].befores;
       tx[ tbl ].afters  = dirac.dals[ tbl ].afters;
+      tx[ tbl ].lock = function(mode, callback) {
```

Comment:
You can only run the lock command in the context of a transaction.

For example
```
# lock table users in access exclusive mode;
ERROR:  LOCK TABLE can only be used in transaction blocks
```

vs scoped within a transaction

```
# begin;
BEGIN
# lock table users in access exclusive mode;
LOCK TABLE
# commit;
COMMIT
```